libs/amplify/.dart_tool/package_config.json

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
Is this not auto generated code fluff for the libs? 

Should we not add a gitignore on the libs directory